packages/webpack/README.md

```diff
@@ -108,7 +108,7 @@ This plugin will skip policy enforcement for such ignored modules.
 
 - [SES lockdown][] must be added to the page without any bundling or transforming for any security guarantees to be sustained.
   - The plugin is attempting to add it as an asset to the compilation for the sake of Developer Experience. `.js` extension is omitted to prevent minification.
-  - Optionally lockdown can be inlined into the bundle files. You need to list the scripts that get to load as the first script on the page to apply lockdown only once when inlined. When you have a single bundle, you just configure a list with one element. It gets more complex with builds for multiple pages. The plugin doesn't attempt to guess where to inline lockdown.
+  - Optionally lockdown can be inlined into the bundle files. You need to match the scripts that get to load as the first script on the page to apply lockdown only once when inlined. When you have a single bundle, you just configure a regex with one unique file name or a `.*`. It gets more complex with builds for multiple pages. The plugin doesn't attempt to guess where to inline lockdown.
```

Comment:
> You need to match the scripts that get to load as the first script on the page to apply lockdown only once when inlined.

Can you rephrase this?

Also, maybe example usage would be helpful, since it's a little complicated.  e.g., given `<script src="bundle-0.js">` and `<script src="bundle-1.js">`, what would be the regex

Reply:
In case only one bundle file results this is trivial. With a setup that has many entries, the regex needs to match all of them. If you're chunking, you're better off configuring webpack to emit the runtime chunk and match that. I was trying to avoid writing all this, but I could.